### DISCUSSION TITLE
.iconify() and .deiconify() don't work in vscode.

### DISCUSSION THREAD

**Discussion by jorgegbsouza** [TomSchimansky/CustomTkinter#1893] [General]
Hello,

I'm using the customtkinter in vscode with Python 3.9.5 and trying to execute this following commands,
![image](https://github.com/TomSchimansky/CustomTkinter/assets/78984109/c1690252-7998-4a36-b6f9-f50e6db34ccb)
and it isn't working, literally do nothing.

Can somebody help me?

**Comment by Adarsh-365** [comment]
what you want to do actually if you want to disappear window then reappear use app.withdraw() then app.deiconify()

**Reply by jorgegbsouza** [reply]
that is it, I want to disappear a window (login screen) and then appear another

**Comment by SarthakTools** [comment]
Just add this between iconify() and deiconify() root.update()
